testing/testcafe/tests/dataGrid/scrolling.ts

```diff
@@ -1044,6 +1044,78 @@ safeSizeTest('The scroll position of a fixed table should be synchronized with t
   }, 'id', 'text'],
 }));
 
+// T1089634
+test('The page should not be changed when hiding/showing the grid view after the data has been edited', async (t) => {
+  // arrange
+  const dataGrid = new DataGrid('#container');
+  const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
+
+  // assert
+  await t
+    .expect(dataGrid.isReady())
+    .ok();
+
+  // act
+  await dataGrid.scrollTo(t, { y: 500 });
+
+  // assert
+  await t
+    .expect(dataGrid.isReady())
+    .ok();
+
+  await takeScreenshot('T1089634-virtual-scrolling-1.png', '#container');
```

Comment:
Let's replace selector with ```dataGrid.element```

```suggestion
  await takeScreenshot('T1089634-virtual-scrolling-1.png', 'dataGrid.element');
```